docs/modules/game-object-creation/overview.md

```diff
@@ -0,0 +1,27 @@
+# GameObject Creation Feature Module
+
+This feature module contains an default implementation of spawning GameObjects for your SpatialOS entities and offers the ability to customize that process.
+
+## Installation
+
+### 1. Add the package
+
+To add this feature module to your project, add the following line to the `dependencies` section in your `workers/my-unity-project/Packages/manifest.json`:
+
+```json
+    "com.improbable.gdk.gameobjectcreation": "file:<path-to-the-gdk>/workers/unity/Packages/com.improbable.gdk.gameobjectcreation",
+```
+
+> **Note:** `<path-to-the-gdk>` is the relative path between your `manifest.json` and the `gdk-for-unity` project.
+
+### 2. Reference the assemblies
+
+The GameObject Creation Feature Modules contains a single [assembly definition file](https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html) which you must reference. This process differs if you have an assembly defintion file in your own project or not.
+
+#### I have an assembly definition file
```

Comment:
should we add something like 
a) I have ..
b) I don't have ...

as it looks from the preview like a new section. That might be different in the actual live docs though

Reply:
We can make it manually bolded section headers instead of `h4`s to make it more clearer a sub header